src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -73,70 +66,74 @@ const ManageUserPage = ({
     >
       <h1 className="govuk-heading-xl">
         <span className="govuk-caption-xl">Manage user</span>
-        {model?.user.name}
+        {user?.name}
       </h1>
 
-      <Formik<FormValues>
-        enableReinitialize
-        initialValues={{
-          userEmail: '',
-        }}
-        onSubmit={handleSubmit}
-        render={_ => {
-          return (
-            <Form id={formId}>
-              <FormFieldset
-                id={`${formId}-details`}
-                legend="Detals"
-                legendSize="m"
-              >
-                <SummaryList>
-                  <SummaryListItem term="Name">
-                    {model?.user.name}
-                  </SummaryListItem>
-                  <SummaryListItem term="Email">
-                    {' '}
-                    {model?.user.email}
-                  </SummaryListItem>
-                  <SummaryListItem term="Phone">-</SummaryListItem>
-                </SummaryList>
-              </FormFieldset>
-              <FormFieldset
-                id={`${formId}-role`}
-                legend="Role"
-                legendSize="m"
-                hint="The users role within the service."
-              >
-                <FormFieldSelect
+      <LoadingSpinner loading={isLoading} text="Loading user">
+        <Formik<FormValues>
+          enableReinitialize
+          initialValues={{
+            userId: userId ?? '',
```

Comment:
This shouldn't be a form value as it's not something the user should be able to change

Reply:
Its not used either so removed